src/Protocol/Email.php

```diff
@@ -62,8 +63,8 @@ public static function connect($mailbox, $username, $password)
 	}
 
 	/**
-	 * @param resource $mbox       mailbox
-	 * @param string   $email_addr email
+	 * @param Connection $mbox       mailbox
```

Comment:
Please add the type hint in the method signature.

Reply:
What do you mean with "method signature"?

Reply:
The method signature is the first line of a method's declaration with the scope, the name and the parameters.

Reply:
So the "method signature" is the function definition? Okay.